### PR TITLE
[w-11670135] Removed power of us url's

### PR DESCRIPTION
**Pull request by mrrigney**
# Critical Changes

# Changes

# Work Items Closed
- [[W-11670135]](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000015HEWtYAO/view)

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
-For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**
+**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in The Power of Us Hub.
```
Comment (aarsvoboda) [inline .github/ISSUE_TEMPLATE.md:1]: @mrrigney I didn't see this one. Line 1: For support questions, please use the K-12 ([Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE).

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+For feature requests, please use the Ideas tab in The Power of Us Hub.**
```
Comment (aarsvoboda) [inline .github/ISSUE_TEMPLATE.md:2]: @mrrigney I would remove this line.

Comment (aarsvoboda) [reply .github/ISSUE_TEMPLATE.md:2]: Line 2, that is.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mrrigney) [reply .github/ISSUE_TEMPLATE.md]: @aarsvoboda I modified this line, can you please verify?  Made you an approver

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (mrrigney) [reply .github/ISSUE_TEMPLATE.md]: @aarsvoboda I modified this line, can you please verify?  Made you an approver

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (aarsvoboda) [reply .github/ISSUE_TEMPLATE.md]: @mrrigney Here is my suggestion for Line 1:
 **Note: FOR INTERNAL USE ONLY. For support questions, please use the [Sector: K-12 Education](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE) group in Trailblazer Community.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
-**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
-For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**
+**Note: FOR INTERNAL USE ONLY. For support questions, please use the K-12 [Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE?tab=discussion).
```
Comment (aarsvoboda) [inline .github/ISSUE_TEMPLATE.md:1]: @mrrigney One small edit:
...please use the [Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE) in Trailblazer Community.

**Review by aarsvoboda — CHANGES_REQUESTED** [review]
@mrrigney  Left a few edits! Let me know if you have any questions. Have a happy day!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mrrigney) [reply .github/ISSUE_TEMPLATE.md]: @aarsvoboda I made this change

File: metadeploy/labels_en.json [hunk 4]
```diff
@@ -9,15 +9,15 @@
...
             "description": "legal text shown in modal dialog"
         },
         "error_message": {
-            "message": "If you experience an issue with the installation, post in the [Sector: K-12 Education group](https://powerofus.force.com/K12_Community).",
+            "message": "If you experience an issue with the installation, post in the Sector: K-12 Education group.",
```
[the remark this answers is not in the document]
Comment (mrrigney) [reply metadeploy/labels_en.json:20]: @Aaron I made this change, please review

[the remark this answers is not in the document]
Comment (mrrigney) [reply metadeploy/labels_en.json]: @aarsvoboda line 20 does have the link, I think this comment thread is just out of date.  Also, the comma has do do with the json formatting and it is ok

[the remark this answers is not in the document]
Comment (aarsvoboda) [reply metadeploy/labels_en.json]: Thanks @mrrigney!

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (aarsvoboda) [reply .github/ISSUE_TEMPLATE.md]: @mrrigney Sorry if this is a "Level 1 Error" but I have refreshed and I'm still seeing a problem in line 1. It says: "For support questions, please use the **K-12** [Sector: K-12 Education group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi7SAE?tab=discussion).

Comment (mrrigney) [reply .github/ISSUE_TEMPLATE.md]: @aarsvoboda try looking at this under the files changed tab, you should see
image

**Review by jcschultz — APPROVED** [review]

**Review by aarsvoboda — APPROVED** [review]

**Pull request merged**